yt_dlp/extractor/bilibili.py

```diff
@@ -68,20 +64,18 @@ class BiliBiliIE(InfoExtractor):
         },
     }, {
         # Tested in BiliBiliBangumiIE
-        'url': 'http://bangumi.bilibili.com/anime/1869/play#40062',
```

Comment:
Does the old URL still work in browser? If so, we want it to keep working in yt-dlp too (redirect by generic is acceptable)

Reply:
> Does the old URL still work in browser? If so, we want it to keep working in yt-dlp too (redirect by generic is acceptable)

old version of url is not working anymore